### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by gersonjeanpierre** [Launch-X-Latam/MisionFrontEnd#25] [Show and tell]
![image](https://user-images.githubusercontent.com/114420460/192407074-c96f4059-95c3-4041-9f15-c0ebc5551d79.png)
![image](https://user-images.githubusercontent.com/114420460/192407164-3d9906c2-0179-4f79-a5bd-b284675e4edd.png)
![image](https://user-images.githubusercontent.com/114420460/192407215-7fe71e95-0138-4397-b83c-5d998d9d4a43.png)

# Hi explorers!
Saquen provecho del contenido que ofrece github con la cuenta institucional @innovaccion.mx.
Y como sabrán todo estos cursos y recursos es en ingles.
Muchas ganas al ingles ya que desde la documentación y buena información están en ese idioma 👍

~~Si les interesa en las proximas horas hare un post de como activar su pack en este post.~~

## Buena suerte en sus viajes 🚀

# Activando tu Student Developer Pack
1. Crear una cuenta github o anexar con el correo institucional @innovaccion.mx
2. Ir al [link](https://education.github.com/) y logearse
3. Ir a la cuarta opción de este menu.
![image](https://user-images.githubusercontent.com/114420460/192412382-69dffed7-8aea-4d6f-8920-da34b00402a1.png)
4. Bajar y escribir "Learn" en la caja de texto y clickear continuar.
![image](https://user-images.githubusercontent.com/114420460/192412626-e8afdf92-3583-414e-83ae-e3982ff5fef3.png)
5. La siguiente pagina les pedirá subir un archivo comprobante en jpg o png. En nuestra caso sera todo el correo que nos enviaron.
6. En su correo le dan a imprimir como aparece en la imagen.
![image](https://user-images.githubusercontent.com/114420460/192412931-9e8568e0-1fc2-4b14-b803-e07fc106ee1c.png)
7. Clickean nuevamente en Imprimir y lo guardan en pdf.
![image](https://user-images.githubusercontent.com/114420460/192413108-daca791c-bd0c-4309-b44f-82921c146e55.png)
8. Transforman todo el pdf a un solo jpg. Pueden usar convertidores online o a puro paint.
![image](https://user-images.githubusercontent.com/114420460/192413215-04e9a612-f0f4-4385-8b92-731cdafe0bdf.png)
9. Suben su jpg o png del correo y le clic en validar. Se tardara unos minutos y luego les dirá que estará en revisión. En mi caso demoro 5 min. Y listo les aparecerá esta confirmación:
![image](https://user-images.githubusercontent.com/114420460/192413537-079bdc00-2ec1-4540-a649-3fdf7008dbad.png)

**Comment by Startupero** [comment]
Hola, me parece interesante algunas de las cosas que ofrecen, aunque la mayoría son tipo cupones o periodos de prueba gratis. 

Creo que vale la pena. Para solicitar te recomiendan un correo relacionado a entidades educativas, en este caso ¿cuál usarías? Mi cuenta de github ya la tenía antes de este programa de LaunchX, si es aprovechar que ya estoy dentro de una organización en github lo lógico seria utilizar el correo con el que me aceptaron en LaunchX.

**Reply by JosafatJimenezB** [reply]
Me parece que con el de launchx no aplica esta promoción ya que se deben enviar datos para comprobar que eres estudiante por tal motivo no agregan este beneficio como en los otros programas de innovaccion

pero para los que son estudiantes aprovechen este pack de recursos

**Reply by gersonjeanpierre** [reply]
Yo lo use con el correo de launch. Cuando te piden validar con una foto solo descarga en pdf del correo donde nos llega la cuenta institucional, lo transformas en jpg y listo en los proximos minutos te aprueban. Yo lo hice hoy mismo.

**Reply by JosafatJimenezB** [reply]
solo que el detalle es que solo lo podrán obtener estudiantes y no todos lo podrán reclamar

**Reply by gersonjeanpierre** [reply]
> solo que el detalle es que solo lo podrán obtener estudiantes y no todos lo podrán reclamar

Checkea el update del post

**Reply by JosafatJimenezB** [reply]
checado, gracias por la info!!!

**Reply by josmanyel** [reply]
> Yo lo use con el correo de launch. Cuando te piden validar con una foto solo descarga en pdf del correo donde nos llega la cuenta institucional, lo transformas en jpg y listo en los proximos minutos te aprueban. Yo lo hice hoy mismo.

Hola, pero como lo hiciste?, podrías ilustrar el proceso, porque yo intenté con el correo de Launch X y me dice que ese correo ya está en uso.

**Reply by rpajares** [reply]
Que ventajas da esta plataforma?, no las veo aun, solo que se puede compartir informacion o codigo, pero que mas hace?

**Comment by GilbertZP2** [comment]
Ojala pueda realizar una guia para poder aprovechar esta gran oportunidad, muchas gracias.

**Reply by gersonjeanpierre** [reply]
Ya esta el update del post

**Comment by OscarTinajero117** [comment]
Tengo un [fork de hace dos años](https://github.com/OscarTinajero117/onboarding-github-innovaccion), si entran al original dice que esta en revisión ya que solo lo permite con correos institucionales.

**Reply by gersonjeanpierre** [reply]
Tiene que ser con el correo @innovaccion.mx pues este correo es institucional y tiene convenio con microsoft. En el post estan los pasos como hoy mismo he accedido al Pack.

**Reply by rpajares** [reply]
el beneficio es solo tener un nombre de dominio y un correo?

**Comment by MCurtis7** [comment]
Por cuánto tiempo se activan los beneficios?

**Reply by OscarTinajero117** [reply]
### GitHub
Hasta que den de baja el correo

### Otros servicios
Cada servicio tiene una fecha, algunos se dan de baja con GitHub

**Reply by JosafatJimenezB** [reply]
Lo tienes que renovar cada año

**Reply by gersonjeanpierre** [reply]
Los beneficios solo se consumen 1 vez. Por ejemplo para FrontEndMasters el cupo solo son 6 meses de acceso y en el uso de los IDE de JetBrains se renueva cada año como el feature pro de la cuenta github.

**Reply by JosafatJimenezB** [reply]
confirmo, algunos mientras seas estudiante renuevas

**Comment by rfponce** [comment]
¿Tenemos un correo por parte de inovaccion? ¿cómo entramos a él? 🤔

**Reply by gjordanra** [reply]
En el correo que te enviaron ellos de confirmación están tus datos, el correo de 4 páginas donde te explican todo lo que debes hacer y como unirte a este grupo

**Comment by gjordanra** [comment]
Yo me registre con el correo de confirmación que nos enviaron, el primero que era más corto

**Comment by rgplazas** [comment]
Gracias

**Comment by maxkario** [comment]
Ya he intentado de todo y no me ha dejado registrarme como estudiante. 
Solo  me dice que cancele la solititud y vuelva a intentarlo. Ya he subido mas de 10 imagenes porque no me deja seleccionar la misma.

**Reply by gersonjeanpierre** [reply]
Crear una cuenta github o anexar con el correo institucional @innovaccion.mx , si es una cuenta personal antigua debes anexar el correo @innovaccion.mx aunque seria mas rapido y practico crear una nueva con @innovaccion.mx
Yo tambien tengo una cuenta personal antigua pero preferi crear esta nueva con el correo que nos brindo Innovacion Virtual.

**Reply by Keystle** [reply]
El tema es que no te deja subir la misma imagen repetidas veces. Fijate que a veces te manda al Teacher pack y no al Student pack. Asegurate bien de eso primero. De ultima renombrá la imagen antes de subirla o hacele algun cambio de tamaño para ver si asi te la toma.

**Reply by JorgeDevmm** [reply]
A mi también me paso eso cuando me marco error al subir la misma imagen, simplemente abre esa imagen y dale guardar como pero en otra ruta, renómbralo y te dejara subirlo

**Reply by maxkario** [reply]
Yo anexe el correo de innovaccion a mi cuenta personal, de lo contrario no podia ingresar como estudiante. Pero el error persiste. 
Por otra parte es posible que no este disponible para mi pais. Otro error que a veces aparecia en los intentos que tuve, fue la ubicación. puede que para mi pais no este disponible el beneficio. 
![Portapapeles01](https://user-images.githubusercontent.com/59313980/192683765-2cc1fee5-6918-477d-946b-cff1326c5830.png)

Ya en ultimas solo me salia el segundo error como lo digo en el primer comentario.

**Reply by fedeminatta** [reply]
me pueden explicar cómo se anexa el correo de innovación a mí correo personal de github?

**Comment by Keystle** [comment]
Mil gracias, ya lo pude reclamar. Tuve que pelearme nomas con convertir todo el pdf a un solo jpg 😂 pero lo logré despues de varios intentos

**Reply by gersonjeanpierre** [reply]
jaja como dije de pdf conviertanlo a jpg ya sea online o a puro paint 😂

**Comment by erykzon** [comment]
Momento de aprovechar

Excelente en ciclos pasados era mas complicado que te dieran el GSDP.

**Comment by themelonkitty** [comment]
Buenísimo, funcionó sin problemas. A los 2 minutos de enviada la solicitud quedó activo. Muchas gracias :)

**Comment by JorgeDevmm** [comment]
Gracias, bro funciono perfectamente

**Comment by luisdsalgado** [comment]
No me funcionó con la imagen del correo, ¿Qué puedo hacer?

**Reply by gersonjeanpierre** [reply]
Tienes que convertir en pdf el correo y luego volverlo un solo jpg

**Reply by luisdsalgado** [reply]
Eso hice y no funcionó. 🥲

**Reply by JorgeDevmm** [reply]
manda la captura del error entre todos te ayudamos, a mi también me mandaba error al inicio con la imagen, le hice una copia y le puse en otra ruta, por que si mandas desde la misma ubicación te manda error así le cambies de nombre

**Reply by luisdsalgado** [reply]
Esto es lo que me aparece
![7D1B570A-4298-4243-9DAE-A67055C2B84A](https://user-images.githubusercontent.com/114312953/194149815-254af9a9-66a8-403d-802b-e097bc897c54.jpeg)

**Reply by JorgeDevmm** [reply]
manda la imagen que usaste

**Reply by gersonjeanpierre** [reply]
> manda la imagen que usaste
![Correo_ Gerson Jean Pierre - Outlook_page-0001 (1)](https://user-images.githubusercontent.com/114420460/194956512-43da5e2a-9b2b-40b8-98a7-9e2838972af9.jpg)

**Reply by gjordanra** [reply]
use el primer email es mas sencillo, el correo donde dicen que fuiste aprobado para el Launch X, este de 4 paginas es muy largo

**Reply by JorgeDevmm** [reply]
> > manda la imagen que usaste
> > ![Correo_ Gerson Jean Pierre - Outlook_page-0001 (1)](https://user-images.githubusercontent.com/114420460/194956512-43da5e2a-9b2b-40b8-98a7-9e2838972af9.jpg)

Gracias bro, pero era para el anterior comentario que mandara la imagen para orientarlo en que falla, que le daba error, yo pude registrarme normal

**Reply by JorgeDevmm** [reply]
> Esto es lo que me aparece ![7D1B570A-4298-4243-9DAE-A67055C2B84A](https://user-images.githubusercontent.com/114312953/194149815-254af9a9-66a8-403d-802b-e097bc897c54.jpeg)

manda la imagen que usastes

**Comment by T0ny-dev** [comment]
buena :D a probarlo

**Comment by Sebastian-Longhitano** [comment]
Para hacer las fotos más facilmente, descarguen los correos electrónicos en formato PDF con la foto en donde te verifican como alumno de Innovación Virtual, y posteriormente conviertanlo a JPG con alguna herramienta online para convertir de PDF a JPG con un click.

Yo use esta: https://smallpdf.com/es/pdf-a-jpg

Saludos cordiales,
